Breadcrumbs.py

```diff
@@ -32,80 +38,169 @@ def get_row_indentation(points, view, tab_size, limit=1e20):
 def is_white_row(view, points):
   return all(view.substr(pt).isspace() for pt in reversed(points))
 
-def get_breadcrumb(view, points, limit):
+def get_breadcrumb(view, points, regex, separator):
   for pt in points:
     ch = view.substr(pt)
     if not ch.isspace():
-      return view.substr(sublime.Region(pt, min(view.line(pt).b, pt + limit))).strip()
+      linestring = view.substr(sublime.Region(pt, min(view.line(pt).b, pt + 500))).strip()
```

Comment:
I'm not sure what specific use cases you have in mind, that would make stripping the trailing spaces problematic.
For example, `"\\s*(?P<name>.*\\S)"` seems to trim leading and trailing spaces.
Could provide some real world example where it is difficult to get the behavior you want?

One more reason I'm agains (r)strip, is that it kinda solves only part of the problem, which is to remove spaces, while in real life, I might as well remove `{` and/or `:`, so I'll need regexp power anyway.